docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common acronyms.
+
+## CustomResourceDefinitions
+
+### CatalogSources
+
+**Definition**: The CatalogSource is a repository of [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and packages that define an application. 
```

Comment:
Do you think rephrasing this definition to 
"A CatalogSource is a repository of operators, where each operator is defined by it's CSVs, CRDs and package.yaml."
will be clearer? 
The way it's currently written,  one might confuse the use of the word `application` for a `CatalogSource`.